draft-irtf-cfrg-opaque.md

```diff
@@ -1608,6 +1608,14 @@ to support implementations which provide mitigations against client enumeration
 - Deserialization checks: When parsing messages that have crossed trust boundaries (e.g.
 a network wire), implementations should properly handle all error conditions covered in
 {{I-D.irtf-cfrg-voprf}} and abort accordingly.
+- Additional client-side entropy: OPAQUE supports the ability to incorporate the
+client identity alongside the password to be input to the OPRF. This provides additional
+client-side entropy which can supplement the entropy that should be introduced by the
+server during an honest execution of the protocol.
+- Client-side authentication of the server: Note that online guessing attacks
+(against any Asymmetric PAKE) can be done from both the client side and the server side.
+This means that additional checks, such as implementing client-side authentication of
```

Comment:
The phrase "client-side authentication of the server" is somewhat confusing to me. Perhaps we can say that the channel over which OPAQUE registration and login is run run is server-authenticated?

Reply:
See my edit and let me know what you think!